src/compiler/commandLineParser.ts

```diff
@@ -215,12 +218,18 @@ const libEntries: [string, string][] = [
     ["es2022.error", "lib.es2022.error.d.ts"],
     ["es2022.intl", "lib.es2022.intl.d.ts"],
     ["es2022.object", "lib.es2022.object.d.ts"],
-    ["es2022.sharedmemory", "lib.es2022.sharedmemory.d.ts"],

```

Comment:
Do we have any tests showing what happens if you were using `"target": "es2022"` or `"lib": ["es2022"]` and `Atomics.waitAsync`?

Reply:
I will add it, as it is not there yet.